Team Gallery page: Restructure in three subsections (#3240)
GenericMappingTools/pygmt: doc/team.md

# PyGMT Team

We are an international team dedicated to building a Pythonic API for the Generic Mapping
Tools (GMT).

All are welcome to become involved with the PyGMT project! For more information about how
to get involved, see the {doc}`contributing`.

Distinguished Contributors are recognized for their substantial contributions to PyGMT,
which may include code, documentation, pull request review, triaging, forum responses,
community building and engagement, outreach, and inclusion and diversity. Maintainers
are recognized for their responsibilities in maintaining the project, as detailed in

> {doc}`maintenance`.

Reviewer: ```suggestion
the {doc}`maintenance`.
```

# PyGMT Team

We are an international team dedicated to building a Pythonic API for the Generic Mapping
Tools (GMT).

All are welcome to become involved with the PyGMT project! For more information about how
to get involved, see the {doc}`contributing`.

> PyGMT Distinguished Contributors are recognized for their substantial contributions to PyGMT,
> which may include code, documentation, pull request review, triaging, forum responses,
> community building and engagement, outreach, and inclusion and diversity. New Distinguished
> Contributors are selected twice per year by those listed below. Distinguished Contributors
> include current or inactive contributors and inactive maintainers, so is not meant as a means
> of conveying responsibilities. Distinguished Contributors who are also active maintainers of
> the PyGMT project have responsibilities detailed in the {doc}`maintenance`.

Reviewer: I am trying to rephrase this paragraph. 

The first paragraph explains the different roles and the second paragraph documents the policy. OK with the changes? Feel free to rephrase in any way you like:
```suggestion
Distinguished Contributors are recognized for their substantial contributions to PyGMT,
which may include code, documentation, pull request review, triaging, forum responses,
community building and engagement, outreach, and inclusion and diversity. Maintainers
are recognized for their responsibilities in maintaining the project, as detailed in
{doc}`maintenance`.

New Distinguished Contributors and Active Maintainers are selected and voted by current
Active Maintainers before each release. Maintainers that are inactive for more than one
year will be moved to Distinguished Contributors.
```

Author: So far this sounds good to me!

## Founders

> :::::{grid} 2 3 3 5

Reviewer: ```suggestion
:::::{grid} 2 3 4 5
```

::::{grid-item-card} Leonardo Uieda

> :margin: 0 3 0 0

Reviewer: ```suggestion
:padding: 1
```
Just tried to play with the `grid-item-card` options. Changing `:margin: 0 3 0 0` to `:padding: 1` gives a more compact layout.

|Old | New |
|---|---|
| <img width="909" alt="image" src="https://github.com/GenericMappingTools/pygmt/assets/3974108/c7ccf71b-122b-469f-9ae2-389fe55cae6d"> |<img width="931" alt="image" src="https://github.com/GenericMappingTools/pygmt/assets/3974108/84532546-8e47-4924-9a60-c0ef2fd9dd7f"> |

Author: Like theis as there all names fit in one line 🙂!

:text-align: center
:img-top: https://avatars.githubusercontent.com/u/290082?v=4

[@leouieda](https://github.com/leouieda)
::::

::::{grid-item-card} Paul Wessel
:margin: 0 3 0 0
:text-align: center
:img-top: https://avatars.githubusercontent.com/u/26473567?v=4

[@PaulWessel](https://github.com/PaulWessel)
::::

:::::

## Active Maintainers

> :::::{grid} 2 3 3 4

Reviewer: ```suggestion
:::::{grid} 2 3 4 5
```

# PyGMT Team

> We are an international team dedicated to building a Pythonic API for the Generic Mapping
> Tools (GMT). Our goal is to improve GMT's accessibility for new and experienced users by
> creating user-friendly interfaces with the GMT C API, supporting rich display in Jupyter
> notebooks, and integrating with the PyData ecosystem.

Reviewer: I feel we can remove the sentence about our goals, because it's already mentioned at https://www.pygmt.org/dev/#project-goals:
```suggestion
We are an international team dedicated to building a Pythonic API for the Generic Mapping
Tools (GMT).
```

All are welcome to become involved with the PyGMT project! For more information about how
to get involved, see the {doc}`contributing`.

PyGMT Distinguished Contributors are recognized for their substantial contributions to PyGMT,
which may include code, documentation, pull request review, triaging, forum responses,
community building and engagement, outreach, and inclusion and diversity. New Distinguished
Contributors are selected twice per year by those listed below. Distinguished Contributors
include current or inactive contributors and inactive maintainers, so is not meant as a means
of conveying responsibilities. Distinguished Contributors who are also active maintainers of
the PyGMT project have responsibilities detailed in the {doc}`maintenance`.

## Founders

> :::::{grid} 2 3 3 4

Reviewer: Maybe five cards per row is better:
```suggestion
:::::{grid} 2 3 3 5
```

:img-top: https://avatars.githubusercontent.com/u/23025878?v=4

[@michaelgrund](https://github.com/michaelgrund)
::::

::::{grid-item-card} Will Schlitzer
:margin: 0 3 0 0
:text-align: center
:img-top: https://avatars.githubusercontent.com/u/29518865?v=4

[@willschlitzer](https://github.com/willschlitzer)
::::

::::{grid-item-card} Yvonne Fröhlich
:margin: 0 3 0 0
:text-align: center
:img-top: https://avatars.githubusercontent.com/u/94163266?v=4

[@yvonnefroehlich](https://github.com/yvonnefroehlich)
::::

:::::

## Distinguished Contributors

> :::::{grid} 2 3 3 4

Reviewer: ```suggestion
:::::{grid} 2 3 4 5
```
